src/query/interpreter.cpp

```diff
@@ -791,6 +791,32 @@ Callback HandleStreamQuery(StreamQuery *stream_query, const Parameters &paramete
   }
 }
 
+Callback HandleConfigQuery() {
+  Callback callback;
+  callback.header = {"name", "default_configuration", "current_configuration"};
+
+  callback.fn = [] {
+    std::vector<GFLAGS_NAMESPACE::CommandLineFlagInfo> flags;
+    GetAllFlags(&flags);
+
+    std::vector<std::vector<TypedValue>> results;
+
+    for (const auto &flag : flags) {
+      if (flag.hidden) continue;
```

Comment:
```suggestion
      if (flag.hidden) {
        continue;
      }
```

In general I think we prefer to write out the braces to make sure we are safe if we want to extend the condition.